roles/ara/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+ ### Update apt cache, but don't mark changed
+ # Works around Ansible 2.2 bug
+ # TODO: Remove with Ansible 2.3
+- name: Update apt cache
+  apt:
+    update_cache: yes
+  changed_when: False
+
+- name: Install deps
+  apt:
+    name: "{{ item }}"
+    state: installed
+  with_items:
+    - gcc 
+    - libffi-dev 
+    - python-dev
+    - libssl-dev
+
+- name: Install ara
+  pip:
+    name: ara
```

Comment:
python-app role should be able to handle this. You don't want to pip install without a virtualenv. It gets messy.